express-notes-api/server.js

```diff
@@ -0,0 +1,100 @@
+import express from 'express';
+import * as fs from 'node:fs';
+import * as fsPromises from 'node:fs/promises';
+
+const fileData = fs.readFileSync('data.json', 'utf8');
+const data = JSON.parse(fileData);
+const notes = data.notes;
+const writeFileDataJSON = () => fsPromises.writeFile('data.json', JSON.stringify(data, null, 2));
```

Comment:
While this works fine for this challenge, it's generally better to read the source data at request time (not startup time) to prevent endpoint from working with out-of-date data.